fluentd/event-exporter/main.go

```diff
@@ -0,0 +1,107 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"net/http"
+	"os"
+	"os/signal"
+	"strings"
+	"syscall"
+	"time"
+
+	"github.com/golang/glog"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/contrib/fluentd/event-exporter/sinks"
+	"k8s.io/contrib/fluentd/event-exporter/sinks/stackdriver"
+)
+
+var (
+	resyncPeriod = flag.Duration("resync-period", 1*time.Minute, "Reflector resync period")
+	sinkType     = flag.String("sink-type", stackdriver.SdSinkName, "Name of the sink "+
```

Comment:
The we will introduce them. It totally doesn't make sense to maintain code "just in case".